examples/Examples.hs

```diff
@@ -19,18 +19,22 @@ import Streamly.Internal.Data.Unfold (Unfold)
 import qualified Streamly.Prelude as S
 import qualified Streamly.Data.Fold as FL
 import qualified Streamly.Data.Array.Storable.Foreign as A
+import qualified Streamly.FileSystem.Handle as FH
+import qualified Streamly.Unicode.Stream as U
 import qualified Streamly.Network.Socket as SK
 import qualified Streamly.Network.Inet.TCP as TCP
 
 import qualified Streamly.Internal.Data.Stream.IsStream as S
-import qualified Streamly.Internal.Unicode.Stream as U
-import qualified Streamly.Internal.Data.Fold as FL
+       (intercalateSuffix)
+import qualified Streamly.Internal.Unicode.Stream as U (words, unwords)
+import qualified Streamly.Internal.Data.Fold as FL (mkPure, classify)
 import qualified Streamly.Internal.Data.Unfold as UF
-import qualified Streamly.Internal.FileSystem.Dir as Dir
+       (fromList, fold, enumerateFromToIntegral, identity, outerProduct, map)
+import qualified Streamly.Internal.FileSystem.Dir as Dir (toFiles)
 import qualified Streamly.Internal.FileSystem.File as File
-import qualified Streamly.Internal.FileSystem.Handle as FH
-import qualified Streamly.Internal.Network.Socket as SK
-import qualified Streamly.Internal.Data.Stream.Parallel as Par
+       (toChunks, fromChunks, toBytes, fromBytes, write, read)
+import qualified Streamly.Internal.Network.Socket as SK (writeStrings)
```

Comment:
We can expand this to its definition and expose everything required for that definition.